src/L2/paused/L2VotingPowerPaused.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.23;
+
+import { Ownable2StepUpgradeable } from "@openzeppelin-upgradeable/contracts/access/Ownable2StepUpgradeable.sol";
+import { Initializable } from "@openzeppelin-upgradeable/contracts/proxy/utils/Initializable.sol";
+import { UUPSUpgradeable } from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+import { IL2LockingPosition } from "../../interfaces/L2/IL2LockingPosition.sol";
+import { L2VotingPower } from "../L2VotingPower.sol";
+
+contract L2VotingPowerPaused is L2VotingPower {
+    /// @notice Setting global params.
+    function initializePaused() public reinitializer(2) {
+        version = "1.0.0-paused";
+    }
+
+    /// @notice Override the modifyLockingPosition function to pause VotingPower interactions.
+    function adjustVotingPower(
```

Comment:
As Matjaz mentioned already, this one should not be paused as it would otherwise also pause almost all staking functionality.

Reply:
updated